.gitignore

```diff
@@ -7,7 +7,6 @@ __pycache__/
 *.so
 
 # Distribution / packaging
-Pipfile.lock
```

Comment:
[We generally don't want to include Pipfile.lock for libraries](https://pipenv-fork.readthedocs.io/en/latest/basics.html#general-recommendations-version-control) because that ties us into dependencies for a specific Python version and potential contributors might not have the same version as us, potentially creating issues for them (and likely making most PRs play a game of cat-and-mouse game of different people re-locking on different versions)

Reply:
Sorry, I see how that was ambiguous. We don't want to include it in version control for projects that aren't pinned to a specific Python version